### PR TITLE
maint(typos): rename PDF to DPF

### PR DESCRIPTION
**Pull request by awoimbee**
Just a small mistake I found in the docs, used [rg](https://github.com/BurntSushi/ripgrep) and [rsr](https://github.com/awoimbee/rsr) to try to fix every misspelling.

BTW in this PR I'm just fixing the project name, but [typos](https://github.com/crate-ci/typos) finds a lot of mistakes, it might be a good idea to add it to your CI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The latest version of DPF supports Ansys solver results files for:
 - CFX (`.cad/dat.cff`, `.flprj`)
 
 For more information on file support, see the [main page](https://dpf.docs.pyansys.com/version/stable/index.html)
-in the PDF-Core documentation.
+in the DPF-Core documentation.
```
Comment (PProfizi) [inline README.md:31]:
```suggestion
in the PyDPF-Core documentation.
```

File: docs/source/user_guide/custom_operators_deps.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@ and reference a folder or ZIP file with the sites of the dependencies in an XML
 located next to the folder for the plug-in package. The XML file must have the same
 name as the plug-in package plus an ``.xml`` extension.
 
-When the :py:func:`ansys.dpf.core.core.load_library` method is called, PDF-Core uses the
+When the :py:func:`ansys.dpf.core.core.load_library` method is called, DPF-Core uses the
```
Comment (PProfizi) [inline docs/source/user_guide/custom_operators_deps.rst:6]:
```suggestion
When the :py:func:`ansys.dpf.core.core.load_library` method is called, PyDPF-Core uses the
```

**Review by PProfizi — APPROVED** [review]
Hi @awoimbee, thank you for correcting these typos.
I do not know `rg` nor `rsr`, we use `flake` and `codespell` in our CI.
The PDF mistakes I guess are due to this word being accepted by default.
As for the rest of the typos, I'll give a try to `rg` and `rsr`. 
Keep in mind though that a good part of our code is generated by the server and may contain spelling mistakes that this repo has no power over, such as in the `ansys.dpf.core.operators` module.
Also just proposing two modifications to the corrections you proposed.
Our CI cannot run on forks so I'll merge manually once this is ready.
Thanks!

**Comment by awoimbee** [comment]
`rg` and `rsr` are only alternatives to `grep` and `sed` really, but I think you should try `typos`, it finds issues in this repo like:
`Runn[-n-]i{+n+}g` `pan[-n-]el` `instan[-c-]{+t+}iate` `nu{+m+}ber` `diction[-n-]ary` `wor{+k+}flow` `cum[-m-]ulative_index` `recombine_harmonic_ind[-e-]{+i+}ces_cyclic` `Syntax[-e-]Error`

**Comment by PProfizi** [comment]
> `rg` and `rsr` are only alternatives to `grep` and `sed` really, but I think you should try `typos`, it finds issues in this repo like: `Runn[-n-]i{+n+}g` `pan[-n-]el` `instan[-c-]{+t+}iate` `nu{+m+}ber` `diction[-n-]ary` `wor{+k+}flow` `cum[-m-]ulative_index` `recombine_harmonic_ind[-e-]{+i+}ces_cyclic` `Syntax[-e-]Error`

Indeed, I see them. I think most of them are in parts of the code generated by the server, meaning we should improve typo detection there. Thank you! I'll merge the PR.

**Pull request merged**